index.html

```diff
@@ -3398,6 +3322,12 @@ <h3>The <dfn>EventSubscriptionHandler</dfn> callback</h3>
                 Create an <a>Event</a> notification |response| according to the
                 <a>Protocol Bindings</a> from |data| and |subscriber|, including
                 its |options|.
+              </li>
+              <li>
+                  If |data| is `undefined`, assume that the notification |response| 
+                  will contain a `empty` data payload as specified by <a>Protocol Bindings</a>. 
```

Comment:
This is too generic here. From an algorithm we need to point to exact steps or algorithms from another spec.
If the referred spec doesn't have that, we need to figure that out for the HTTP protocol binding and include the steps here. Let's track that in a separate issue.

Reply:
see https://github.com/w3c/wot-scripting-api/issues/430